go.mod

```diff
@@ -1,6 +1,6 @@
 module istio.io/istio
 
-go 1.22.0
+go 1.23.0
```

Comment:
Not sure if we typically update the minor go version in patch releases. Could you confirm this?

Reply:
This is just bumping the min version for builds, which was required by one of the dependency upgrades. On `release-1.24` this was fine, as it was already being built with Go 1.24, but for the `release-1.23` branch in this PR, this depends on https://github.com/istio/tools/pull/3160

Reply:
Blocked pending update in https://github.com/istio/tools/pull/3163

Reply:
Waiting on automator PR to open in https://github.com/istio/test-infra/pulls after merge of https://github.com/istio/common-files/pull/1162